### DISCUSSION TITLE
using GLMsingle with jittered inter-trial intervals

### DISCUSSION THREAD

**Discussion by jonathanwehnert** [cvnlab/GLMsingle#84] [Q&A] [answered]
Hi! I noticed in the Wiki's FAQ on experimental conditions with different durations [https://glmsingle.readthedocs.io/en/latest/wiki.html#how-do-i-deal-with-experimental-conditions-with-different-durations](url) that "GLMsingle is not particularly suited to" estimating betas of trials with different timecourses. Does this apply in designs with jittered inter-trial intervals (ITI)? 
We were planning on randomly jittered ITIs ranging from 2 to 8 seconds (_M_ = 3 s) in our experiment. Would this be a problem with GLMsingle? 
If so, would you recommend just not jittering the ITI and instead using a consistent 3 s ITI? (Stimulus duration is ~1 s.)
Or would it be sufficient to pseudorandomize jittered ITIs so that for each condition, the mean ITI = 3s?

**Comment by kendrickkay** [comment, marked as answer]
Stimulus event duration is distinct from the time in between those events (e.g. ITI).  The FAQ is referring to event durations, not ITIs.  Your design in which ITIs are jittered should be fine.  The only thing to watch out for is making sure you can reasonably code your event onsets at the granularity of the fMRI volumes (e.g. TRs).

**Reply by jonathanwehnert** [reply]
oh wow, I completely misread that FAQ section. Thank you for clarifying!
